shell/platform/android/test/io/flutter/embedding/engine/systemchannels/KeyboardChannelTest.java

```diff
@@ -32,6 +32,8 @@ private static BinaryMessenger.BinaryReply sendToBinaryMessageHandler(
     return reply;
   }
 
+  @SuppressWarnings("deprecation")
+  // setMessageHandler is deprecated.
```

Comment:
Is there an issue we can reference for this? If we have a test verifying that our own code is calling our own deprecated method, it seems like there's a problem somewhere.